### ISSUE TITLE
TIGER Data Layer Not Appearing

### ISSUE THREAD

**Issue by CurlingMan13** [openstreetmap/iD#9920]
### How to reproduce the issue?

1) Edit using ID editor
2) under Background Settings > Overlays, select TIGER 2017 to TIGER 2022.

TIGER overlay does not appear, even where it previously existed.

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.27.1

### Which browsers are you seeing this problem on?

Chrome

**Comment by nickrsan** [comment]
As a heads up, I suspect it's related to this change and will require a minor change to iD and a new release to fix: https://community.openstreetmap.org/t/updating-the-url-for-tiger-road-overlay-layers/103880

**Comment by nickrsan** [comment]
Also, related discussion in the OpenStreetMap US Slack for members (or anyone wanting to join): https://osmus.slack.com/archives/C029HV951/p1695398195227329

TL;DR: Also happening in JOSM. The ticket for JOSM is here: https://josm.openstreetmap.de/ticket/23198#ticket though it may have been resolved now

**Comment by pathmapper** [comment]
>  will require a minor change to iD and a new release to fix:

Yes, iD needs to include the latest [editor-layer-index](https://github.com/osmlab/editor-layer-index) where this is already fixed (https://github.com/osmlab/editor-layer-index/pull/2060) and publish a new release. 

EDIT: According to https://github.com/openstreetmap/iD/pull/9652#issuecomment-1563236239, including the layer index is done during the release process.

**Comment by bborkmiller** [comment]
The recent releases appear to have addressed this issue. I can get the TIGER layers to load in iD 2.27.3

**Comment by nickrsan** [comment]
Looks like this was resolved in 01f00b9e6b9a6cfed557f86f01612ffa8ed709ad. Closing. Thank you!
